scripts/exregional_aqm_ics.sh.issue926

```diff
@@ -0,0 +1,221 @@
+#!/bin/bash
```

Comment:
@JianpingHuang-NOAA, what is this script (*.issue926?) for?

Reply:
@chan-hoo  It was an accidental addition. It has been deleted.